kafka/adminapi.go

```diff
@@ -639,6 +657,10 @@ func (a *AdminClient) cToTopicResults(cTopicRes **C.rd_kafka_topic_result_t, cCn
 // cConfigResourceToResult converts a C ConfigResource result array to Go ConfigResourceResult
 func (a *AdminClient) cConfigResourceToResult(cRes **C.rd_kafka_ConfigResource_t, cCnt C.size_t) (result []ConfigResourceResult, err error) {
 
+	if a.verifyAdminClient() == false {
```

Comment:
Minor - for this and everywhere - maybe `!verifyClient`is better than an explicit comparison to `false`.